.circleci/config.yml

```diff
@@ -425,10 +444,13 @@ jobs:
       - attach_workspace:
           at: /home/circleci/
       - restore_cache:
-          key: flowmachine-deps-2-{{ checksum "Pipfile.lock" }}
+          key: flowmachine-deps-3-{{ checksum "Pipfile.lock" }}
       - run:
           name: Install graphviz
-          command: sudo apt-get install -y xvfb libgraphviz-dev graphviz
+          command: sudo apt-get update && sudo apt-get install -y xvfb libgraphviz-dev graphviz
+      - run:
+          name: Install python dependencies
+          command: pipenv install --dev --deploy && pipenv run pip install -e . && pipenv install pygraphviz 
```

Comment:
Shouldn't need this because the `install_flowmachine_deps` job creates the venv.

Reply:
I'll pull it out (think a wild sed put it in there)

Reply:
On investigation, it's there because the venv doesn't pick up Graphviz otherwise

Reply:
Ah, my earlier correction should've been `pipenv run pip install` I think, cos otherwise it installs it to system.

Reply:
Works with `pipenv run pip install` after some invalidation of cacheing